ml_model/src/main/java/edu/wm/sealab/featureextraction/Parser.java

```diff
@@ -73,7 +100,27 @@ public static void main(String[] args) {
                 // Extract syntactic features (non JavaParser extraction)
                 SyntacticFeatureExtractor sfe =
                     new SyntacticFeatureExtractor(featureVisitor.getFeatures());
-                Features features = sfe.extract(cuNoComm.toString());
+                    // String methodbody = cuNoComm.getMethod();
+
+                    //NEW CODE -- makes extracts the method in the snippet
```

Comment:
Comments like "NEW CODE" aren't helpful, because they cease to be accurate as soon as this code is merged (at which point it is no longer new). Generally, comments should be timeless: they should avoid referring to "the previous version of the code" or similar, too, for the same reason: the code around them is likely to change. Remember that the main audience for a comment is a reader of the code months or years in the future.

Reply:
This was code that I was told to copy/paste in; I'll remove that comment.

Reply:
@Allusionist8359 As I mentioned in an earlier comment please pay extra attention when taking something from somewhere else.